internal/cmd/login.go

```diff
@@ -113,3 +117,36 @@ func pingAPI(ctx context.Context, workspaceURL *url.URL, token string) error {
 	}
 	return nil
 }
+
+// isWSL determines if coder-cli is running within Windows Subsystem for Linux
+func isWSL() (bool, error) {
+	if runtime.GOOS == "darwin" || runtime.GOOS == "windows" {
```

Comment:
it looks like we should make use of constants that are already defined:

https://github.com/cdr/coder-cli/blob/582f213d183be7135e2fcf0e6f839ea320e6c2f3/internal/cmd/update.go#L37

we should add `goosDarwin` as a const as well

Reply:
Thank you for your review. Updated!

Reply:
This may resolve  → https://github.com/cdr/coder-cli/pull/448